### PR TITLE
Ticket 28 :  build table 

### PR DESCRIPTION
**Pull request by maximFeldman1**

### CODE REVIEW

File: src/Components/ProductsList/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { ProductService } from "../../shared/services/product-service";
+import { useQuery } from "react-query";
+import React, { useCallback, useMemo, useRef, useState } from "react";
+import { render } from "react-dom";
+import { AgGridReact } from "@ag-grid-community/react";
+// import "@ag-grid-community/styles/ag-grid.css";
+// import "@ag-grid-community/styles/ag-theme-alpine.css";
```
Comment (itairom-appsforce) [inline src/Components/ProductsList/index.tsx:7]: remove comments

File: src/Components/ProductsList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  );
+  const gridStyle = useMemo(() => ({ height: "100vh", width: "100vw" }), []);
+
+  const [columnDefs, setColumnDefs] = useState<ColDef[]>([
+    { headerName: "ID", valueGetter: "node.id" },
+    { field: "name" },
+    { field: "brand" },
+    { field: "price" },
+    { field: "image" },
+  ]);
```
Comment (itairom-appsforce) [inline src/Components/ProductsList/index.tsx]: move to separate file

File: src/Pages/Products/index.tsx [hunk 3]
```diff
@@ -1,12 +1,37 @@
-import React, { useEffect } from "react";
-import { apiUrl } from "../../shared";
 import { ProductService } from "../../shared/services/product-service";
 import { useQuery } from "react-query";
+import React, { useCallback, useMemo, useRef, useState } from "react";
+import { render } from "react-dom";
+import { AgGridReact } from "@ag-grid-community/react";
+// import "@ag-grid-community/styles/ag-grid.css";
+// import "@ag-grid-community/styles/ag-theme-alpine.css";
```
Comment (itairom-appsforce) [inline src/Pages/Products/index.tsx]: remove

File: src/Components/ProductsList/index.tsx (hunk 1 above)
Comment (maximFeldman1) [reply src/Components/ProductsList/index.tsx:7]: Fixed

File: src/Components/ProductsList/index.tsx (hunk 2 above)
Comment (maximFeldman1) [reply src/Components/ProductsList/index.tsx]: Fixed

**Pull request merged**